### PR TITLE
add open api Cross domain

### PR DESCRIPTION
**Pull request by xilipilige**
Please do not create a Pull Request without creating an issue first.

## What is the purpose of the change

add cors support
issue #1983

## Brief changelog

add file    com.alibaba.nacos.console.config.CorsConfig.java
modify file     com.alibaba.nacos.console.config.WebSecurityConfig.java

### CODE REVIEW

File: console/src/main/java/com/alibaba/nacos/console/config/CorsConfig.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    @Bean
+    public CorsFilter corsFilter() {
+        UrlBasedCorsConfigurationSource source = new UrlBasedCorsConfigurationSource();
+        CorsConfiguration config = new CorsConfiguration();
+        config.setAllowCredentials(true);
+        config.addAllowedOrigin("*");
+        config.addAllowedHeader("*");
+        config.setMaxAge(18000L);
+        config.addAllowedMethod("*");
+        return new CorsFilter(source);
```
Comment (nkorange) [inline console/src/main/java/com/alibaba/nacos/console/config/CorsConfig.java]: You missed this:
```java
 source.registerCorsConfiguration("/**", config);
```

Comment (xilipilige) [reply console/src/main/java/com/alibaba/nacos/console/config/CorsConfig.java]: hi @nkorange 
Will this CORS feature be added to Nacos? If so, I will submit a new PR

Comment (nkorange) [reply console/src/main/java/com/alibaba/nacos/console/config/CorsConfig.java]: > hi @nkorange
> Will this CORS feature be added to Nacos? If so, I will submit a new PR

Yes, and you can just commit new changes in this PR.

Comment (xilipilige) [reply console/src/main/java/com/alibaba/nacos/console/config/CorsConfig.java]: my new PR is #2116  please review the code

**Pull request closed without merging**